app/services/foreman/parameters/caster.rb

```diff
@@ -110,11 +110,15 @@ def cast_yaml
       end
 
       def load_yaml_or_json
+        self.class.load_yaml_or_json(value)
+      end
+
+      def self.load_yaml_or_json(value)
```

Comment:
This can stay as an instance method?  It doesn't appear to be called as a class method anywhere.